Day-4/src/components/Header.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import "./styles.css";
+
+const Header = ({ members }) => {
+  return (
+    <div className="heading">
+      <h1>Welcome to Avengers Team List</h1>
```

Comment:
Liked the idea of rendering the length of data too.

Reply:
Thank you ❤️